package.json

```diff
@@ -69,6 +69,7 @@
         "kea-localstorage": "^1.1.1",
         "kea-router": "^1.0.0",
         "kea-window-values": "^0.0.1",
+        "lodash": "^4.17.21",
```

Comment:
@mariusandra - I have no idea if this is a good idea. This was blocking `yarn build` which was failing without it. Let me know if there is some other better fix here

Reply:
Very strange. We don't use this library directly in our code. What was the error?

Reply:
weird...I removed it and it's working fine now. 🤷‍♂️